test/test_gradient_lsrtm.jl

```diff
@@ -84,19 +78,22 @@ dobs0 = F0*q
 		end
 
 		# test that with zero dm we get the same as fwi_objective for residual
-		ENV["OMP_NUM_THREADS"]=1
+		ENV["OMP_NUM_THREADS"] = 1
 		Jls, gradls = lsrtm_objective(model0, q, dobs, 0f0.*dm; options=opt, nlind=true)
 		Jfwi, gradfwi = fwi_objective(model0, q, dobs; options=opt)
 		@test isapprox(Jls, Jfwi; rtol=0f0, atol=0f0)
 		@test isapprox(gradls, gradfwi; rtol=0f0, atol=0f0)
 	end
 end
 
+
 # Test if lsrtm_objective produces the same value/gradient as is done by the correct algebra
-cases = [(true, false, true), (false, false, true), (true, true, false), (true, false, false), (false, true, false), (false, false, false)]	# DFT and optimal_checkpointing normally don't co-exist
-@testset "LSRTM gradient linear algebra test with $(nlayer) layers and tti $(tti) and freesurface $(fs) and isic $(isic) and optimal_checkpointing $(optchk) and DFT $(dft)" for (isic,optchk,dft) = cases
+@testset "LSRTM gradient linear algebra test with $(nlayer) layers, tti $(tti), freesurface $(fs)" begin
```

Comment:
add in-place fwi_objective, lsrtm_objective, twri_objective test if not overkill

Reply:
It is overkill!! And it is already tested since the non-inplace calls it there is no benefit in re-testing it.